apps/condo/domains/property/components/PropertyAddressSearchInput.tsx

```diff
@@ -41,9 +41,19 @@ export const PropertyAddressSearchInput: React.FC<IAddressSearchInput> = (props)
 
     const searchAddress = useCallback(
         (query, skip) => {
+            const splitted_query = query.split(query_split_regex)
+            let and_sequence = {}
+            splitted_query.forEach((element) => {
+                and_sequence = {
+                    AND: {
+                        address_contains_i: element,
+                        ...and_sequence,
+                    },
```

Comment:
It looks too complex! try to find simple solution

Reply:
It is the only one solution. GraphQL can't work with more than one AND clause in request, so we pack AND clause to AND clause